File: critter_storage/premade/Food.java
package critter_storage.premade;
import game.*;
import java.awt.*;

public class Food extends Critter {
    public Action getMove(CritterInfo info) {
        return Action.INFECT;
    }

    public Color getColor() {
        return Color.GREEN;
    }

    public String toString() {
        return "F";
    }
    public String getCritterName(){
        return "Food";

    }

}

File: CSharpVersion/CritterStorage/Premade/Food.cs
using System.Drawing;
using CritterSimulator.Game;

namespace CritterSimulator.CritterStorage.Premade
{
    public class Food : Critter
    {
        public override Action GetMove(ICritterInfo info)
        {
            return Action.Infect;
        }

        public override Color GetColor()
        {
            return Color.Green;
        }

        public override string ToString()
        {
            return "F";
        }
    }
}
